assets/payment/settings_script.js

```diff
@@ -51,6 +51,18 @@ function customStripeForm(planRank){
 	}
 	showCustomFormWindow(planRank, CurrentWindow);
 }
+//this function is used by the dialog CustomFormWindow 
+function thankYouPage(mode){
```

Comment:
Codacy found an issue: ['thankYouPage' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5975032)